### ISSUE TITLE
I can't get the spell checker to work

### ISSUE THREAD

**Issue by axelsemtwork** [AnKing-VIP/ankiweb-placeholder#83]
#### Checklist

*Please replace the space inside the brackets with an **x** if the following items apply:*

 - [ x] I've verified that I use the latest version of the add-on by redownloading it from AnkiWeb
 - [ x] I've checked if anyone else asked this question before by looking through the issue reports.

#### What Anki version are you using?

Version ⁨2.1.65 (aa9a734f)⁩
Python 3.9.15 Qt 5.15.2 PyQt 5.15.5

*Please open Anki, go to Help → About (windows) or Anki → About Anki (mac) to find the version. Edit the value between the backticks below

```
2.1.65
```

#### Your question

*A clear and concise question about the add-on.*

I've got the add-on downloaded, but the only thing that seems to be working is that it's underlining words in red- but when I click the word there's not a place to select the correct spelling. Am I missing something obvious?

**Comment by AnKingMed** [comment]
That is odd.. I've had a lot of people have issues on 2.1.65 with various addons. Can you try on 2.1.64 or 23.10+?

What dictionaries are you using? You can see it in the config settings (AnKing->Spell checker)
![image](https://github.com/AnKing-VIP/ankiweb-placeholder/assets/57678116/c165a1f0-dbb5-463a-929b-cba445a38329)

**Comment by axelsemtwork** [comment]
![image](https://github.com/AnKing-VIP/ankiweb-placeholder/assets/159754698/1c7e5877-39d3-4df4-bfe7-74c19d0e1e12)

**Comment by AnKingMed** [comment]
yeah that's odd. Working fine for me. Would you mind trying it on a different Anki version?
